### PR TITLE
Rspec config gem factory_bot_rails database_cleaner capybara

### PR DESCRIPTION
**Pull request by haupt-2855**
## Related Tickets
- [#45392](https://edu-redmine.sun-asterisk.vn/issues/45392)

## WHAT (optional)
- Update gem factory_bot_rails
- Update gem database_cleaner
- Update gem capybara

## Evidence (
Screenshot from 2022-01-24 22-59-46

### CODE REVIEW

**Comment by haupt-2855** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -29,6 +29,7 @@ gem "webpacker", "~> 5.0"
 
 group :development, :test do
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+  gem "factory_bot_rails"
```
Comment (vanvtt-0952) [inline Gemfile]: Cái này cũng ở trong group test là dc ròi cu hì

File: spec/factories/products.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :product do
+    name { "first product" }
```
Comment (vanvtt-0952) [inline spec/factories/products.rb]: dùng faker luôn cu ơi

**Comment by haupt-2855** [comment]
ready

**Pull request merged**